docs/competition/matches/match-scoring.md

```diff
@@ -11,47 +11,76 @@ Any roles related to scoring matches go here.
+- When a robot is causing **significant** damage to itself or another robot, i.e. Mashing into another robot, stopping it from being able to function, or dragging it around without being able to be freed, or overheating a motor by stalling against a wall. It's up to you to decide which robot should be switched off, but their match shouldn't be forfeited. If any damage seems intentional, please report it to the head judge. A checklist for robot damage is available below.
 
 If you have any questions you may first ask the head match scorer and secondly the judge.
 
 ### Template Arena Setup Checklist
 
-1. Tokens in the arena aren't damaged. If they are, swap them out with the spares.
+1. Props (e.g: tokens) in the arena aren't damaged. If they are, swap them out with the spares.
 2. The tokens in the arena are intended for that arena -- this can be established by reading the grey text printed on each marker on the token.
    If there is more than one arena then the markers on the sides of any tokens will have the name of the arena as part of the text in the bottom left of the marker.
 3. There must be the correct number of tokens of each type in the arena. The precise configuration depends on the game, though the per-year version of this checklist should be explicit about what's needed (and not rely on volunteers remembering the rules)
 4. Each token is placed on a suitable token starting positions that are marked out on the arena floor. Where possible these should be colour-coordinated.
 
-![Arena Diagram](https://cloud.githubusercontent.com/assets/510498/14783612/a9026b9a-0ae8-11e6-89ef-678acd9ec48a.png)
+![Arena Diagram](https://user-images.githubusercontent.com/2551763/55267652-ccb33200-5283-11e9-8a5b-68ffc4026516.png)
 
 ### Robot Damage Checklist
 
 - If the damage is intentional, seek the judge.
 - If the damage is unintentional:
   - If a robot mashes into another robot, but only briefly, leaving both robots free to move (including situations in which, say, one robot has a fragile, flaily arm that gets snapped off in the collision) then you must leave the robots to continue -- the teams are aware that accidental bumps and scrapes are inevitable.
-  - If a robot mashes into another robot, and pins it, stopping it from being able to function (including one dragging the other around if they become intertwined) then at least one of the robots must be switched off, depending on which robot caused the damage, but the scores are **not** forfeited in this case.
+  - If a robot mashes into another robot, and pins it, stopping it from being able to function (including one dragging the other around if they become strongly intertwined) then at least one of the robots must be switched off, depending on which robot caused the damage, but the scores are **not** forfeited in this case.
```

Comment:
Is this really the case? I've definitely not seen this happen before

Reply:
I guess the emphasis here should be that it actually damages the other robot, I'll revise it

Reply:
perhaps "*not*" -> "*not necessarily*"?
We may choose to also disqualify one of the robots, however I think that this is right -- that turning of the robots in this case doesn't necessarily result in a disqualification.

Reply:
Fixed in 613708d